### ISSUE TITLE
Alt text for images

### ISSUE THREAD

**Issue by argyleink** [GoogleChrome/devsummit#246]
Would love to get proper copy for all of our images

**Comment by AlokTakshak** [comment]
@argyleink  I would like to contribute to it

**Comment by vschupmann** [comment]
LOL!  I'll accept anyway. 😄 

Please point me to the images and the best method for supplying the ALT text (GH, email, Jake's content doc, etc.).

**Comment by argyleink** [comment]
i'm thinking out content doc is the best place: it's currently got references to the images already, seems like just adding a doc comment with the alt text should be fine. lets keep alt text content with the other content?

**Comment by vschupmann** [comment]
Keeping things together in the Content Doc is perfect. Fewer docs to keep track of.

**Comment by argyleink** [comment]
also, can we include alt text for icons too, i'd like those to properly announce themselves when/if focused

**Comment by vschupmann** [comment]
Sure. Are you referring to icons such as the Twitter icon, Github cat icon, envelope for email?

**Comment by argyleink** [comment]
yep! there's more icons comin, things like add notifications, calendar, time, etc.

**Comment by jakearchibald** [comment]
@vschupmann don't worry about this, I'll do it.

All: I think ensuring images have alt text is something frontend developers can handle, and we'll only bounce it back to content folks if we can't figure out what the alt content should be.

**Comment by vschupmann** [comment]
Okay, Jake! Please let me know if you need anything.
